audio/osc2midi/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.1
+PortGroup               github 1.0
+
+github.setup            ssj71 OSC2MIDI 0.2.3
+master_sites            https://github.com/ssj71/OSC2MIDI/archive/
```

Comment:
The "/archive/" part of that path seems to be wrong.

Reply:
I find these urls by going to the releases page and looking at the url for the last tar.gz release. That was it on that page? It does work. I can't make the github PG cough up that url, tho...

Reply:
I wonder if removing master_sites and adding the "v" at the end of github.setup simultaneously would work for you? But if it really doesn't then we do what we have to.

Reply:
Of course I started that way, and went to this when it didn't work.

give it a try! maybe I need to add witchazel or eye-of-toad :)

Reply:
That might be your problem. Eye of Newt is usually more effective. But if you've done everything sane then I have no clue.